Resources/public/js/views/ez-contenttypeselectorview.js

```diff
@@ -192,6 +192,13 @@ YUI.add('ez-contenttypeselectorview', function (Y) {
                 return t.get('id') === typeId;
             });
 
+            /**
+             * Fired when creating a new content of a given type
+             *
+             * @event createContent
+             * @param {Object} contentType
+             * @param {String} languageCode - deprecated
```

Comment:
there's no standard way to document that but I guess can be a bit more precise here with something like:
`@param {String} languageCode (always set to "eng-GB"). This parameter is deprecated and will be removed in PlatformUI 2.0`